src/extension/ios/plistBuddy.ts

```diff
@@ -57,7 +58,8 @@ export class PlistBuddy {
                         sdkType
                     );
 
-                    executableList.push(`${scheme}.app`);
+                    const appName = getFileNameWithoutExtension(projectWorkspaceConfigName) + ".app";
+                    executableList.push(appName);
```

Comment:
```suggestion
                    executableList.push(`${scheme}.app`);
```

I am not super fluent in the intricacies of building the react-native apps, and I might just have a weird setup on my side, but I think changing this `executableList` to be built up with the schemes would fix my problems, along with the changes that were already made in this PR to the `projectWorkspaceConfigName`

Reply:
@andyeskridge thanks for your review, it could be that we missed something. I’ll look at the changes more carefully.
Could you please tell us the sequence of steps to reproduce this problem situation? It also will be very helpful for us if you send us a demo project, so we could investigate the problem faster.